src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,7 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        System.out.println(lottery.getRandomBall().toString());
```

Comment:
To test the program, in main() method, create **three** balls using getRandomBall() method from Lottery class. (we can use a cycle)
and do we need `toString` here?